### ISSUE TITLE
The form is blocked for large "Na jak długo udostępniasz nocleg?" values

### ISSUE THREAD

**Issue by ludwiktrammer** [tadeoos/house-of-refuge#31]
Putting any value that's not very small in the "Na jak długo udostępniasz nocleg? (liczba dni)" field of the https://grupazasoby.pl/share form causes the form to be blocked from sending, without presenting any error to the user - the form fails to be submitted, but there are no hints why. Example value that causes the problem: 365.

The problem seems to be occurring on the frontend.

**Comment by tadeoos** [comment]
This sounds serious.. Thanks for reporting!

**Comment by tadeoos** [comment]
ok, the max for input of type number is set to 100, I'm gonna bump it to 1000

**Comment by ludwiktrammer** [comment]
That's more of a workaround than a solution, isn't it? This form is open to the general public and I can still see someone thinking "I don't have any particular limits" and putting in a very large number to reflect that.

I think to properly solve this we would need to either:
1. Add a clear validation message explaining what happened ("The number has to be between X and Y") - right now it looks like the site broke since there is no message
2. Remove the max limit altogether. Is it really needed?

**Comment by tadeoos** [comment]
You're right that It is a workaround. I think I prefer the first solution -- this is also more general problem of handling different kind of errors from the server or improving the validation on the fe (max length for char fields for example is not handled).

**Comment by galczo5** [comment]
I will implement additional validation soon.

**Comment by tadeoos** [comment]
fixed in #41
